#goal definition
int32 goal_id
geometry_msgs/Pose goal
---
#result definition
bool success
---
#feedback
